src/Donations/resources/DonationDetails/app/components/DonorDetails/index.tsx

```diff
@@ -0,0 +1,141 @@
+import {useState} from 'react';
+import {__} from '@wordpress/i18n';
+import {useWatch} from 'react-hook-form';
+
+import SectionHeader, {DropdownTitle, HeaderLink} from '@givewp/components/AdminUI/SectionHeader';
+import {AsyncSelectDropdownField, DisabledTextField} from '@givewp/components/AdminUI/FormElements';
+import {FieldsetContainer} from '@givewp/components/AdminUI/ContainerLayout';
+import {useGetRequest} from '@givewp/components/AdminUI/api';
+import useDebounce from '@givewp/components/ListTable/hooks/useDebounce';
+
+import {StyleConfig} from './StyleConfig';
+import {apiNonce, apiRoot} from '../../../../window';
+
+/**
+ *
+ * @unreleased
+ */
+const endpoint = `${apiRoot.split('/donation')[0]}/donors`;
```

Comment:
@pauloiankoski Do we plan on updating this in a different PR when we upgrade the API root settings?